### ISSUE TITLE
REPL is not available anymore on the website

### ISSUE THREAD

**Issue by xtuc** [fable-compiler/fable-compiler.github.io#8]
See [REPL](http://fable.io/repl.html)

**Comment by xtuc** [comment]
Nevermind it's available at a different URL. Google search is not up-to-date.

**Comment by alfonsogarciacaro** [comment]
Ah, sorry. Yeah, the URL has changed from `fable.io/repl.html` to `fable.io/repl`. There's a link now in the navbar of fable.io main pages (but not on the REPL itself, that's something I still need to change).

**Comment by xtuc** [comment]
The issue is when a user comes from Google since they still show the old URL. Could you maybe add a redirection `fable.io/repl.html` -> `fable.io/repl`?

**Comment by alfonsogarciacaro** [comment]
Good point! I gave it a try and unfortunately it's a bit tricky. The web is hosted on Github so I can make redirects using a Jekyll plugin but this is not working well with the local server I'm using for development. I tried to do it with JS (setting `window.location`) but this caused an infinite loop with urls like `/repl.html` and `/repl`.

I'll try to check later if there's a cleaner solution. Please tell me if you have any other idea :+1:

**Comment by xtuc** [comment]
Is /repl and /repl.html the same page? I would create a new page with just a direction to the new one.

I re-opened and re-purposed this issue.

**Comment by alfonsogarciacaro** [comment]
In my local server [I did that and it worked well](https://github.com/fable-compiler/fable-compiler.github.io/blob/68b1fd383c1229b4d28d7c5015b62bded88cd123/files/redirect.hbs). But when I pushed the changes, apparently the GH server when you look for `fable.io/repl` apparently it checks first if there's a `fable.io/repl.html`, it finds it, the it's redirected to `fable.io/repl`, then checks again for `fable.io/repl.html`... trapped in that loop forever :/

**Comment by xtuc** [comment]
Ok, note that it's the same with the blog.

![screenshot from 2017-09-08 15-04-41](https://user-images.githubusercontent.com/1493671/30212903-1d35b758-94a7-11e7-94f3-c9777f2c888b.png)

**Comment by alfonsogarciacaro** [comment]
Yes, it's the same for the main pages. The blog is even worse, because it's temporarily removed, I have to re-add it :)
